.gitignore

```diff
@@ -1,3 +1,6 @@
+# Example Data
+examples/scikit-learn-example/
```

Comment:
Can you add this entry to the [.gitignore file within the examples directory](https://github.com/XAITK/xaitk-saliency/blob/master/examples/.gitignore) rather than the overall repo? i.e. add a new line like:

```
/scikit-learn-example/**
```